library/build.gradle

```diff
@@ -14,7 +14,7 @@ dokkaGfmPartial {
 
 ktlint {
     filter {
-        exclude { it.file.path.contains("xmtp_dh") }
```

Comment:
That's a good reminder. I believe `libxmtp` now has a full set of replacement APIs for what you're using in `xmtp_dh` (if anything is missing we can add it). Maybe an item to add to the backlog for moving over to a single Rust dependency just like iOS did. Obviously not critical.

Reply:
O good call out. I'm only using 2 rust calls in V2 so I can probably clean those up

Reply:
Okay! I was able to remove all the v2 rust stuff 🎉